src/hooks/useActiveDescendantListFocus/index.js

```diff
@@ -0,0 +1,157 @@
+import { useCallback, useMemo, useRef, useState } from "react";
+import useKeyEvent from "../useKeyEvent";
+import useEventListener from "../useEventListener";
+
+const ARROW_DIRECTIONS = {
+  UP: "ArrowUp",
+  DOWN: "ArrowDown",
+  RIGHT: "ArrowRight",
+  LEFT: "ArrowLeft"
+};
+
+const ENTER_KEYS = ["Enter"];
```

Comment:
What about space? is it considered a selection as well?
If so, consider using `SELECTION_KEYS` from `src/hooks/useFullKeyboardListeners.js`

Reply:
it was not supported in the combobox until now but I can add it :)

Reply:
Ok it problematic because it make spaces to not work on the search box. I am there is a way to fix it but I prefer to fix it in a different PR (out of my scope and capacity... and until now it was not supported)

Reply:
sure! it was just an idea anyway :D